Kritika_Sachdeva/Calculator/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Calculator</title>
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+
+    
+    <link rel="stylesheet" type="text/css" href="./styles.css">
+<!--<script src="./script.js" ></script> -->
+
+
+</head>
+<body>
+    
+    <div class="container">
+        <div class="scrn"><form name="calc"><input type="text" name="display" id="scn" size="20" readonly></form></div>
+
+        <div class="calc_btns">
+            <p>
+                <input type="button" value="x!" class="other_opr" onclick=fact();>
+                <input type="button" value="C" class="other_opr" onclick="calc.display.value=''">
```

Comment:
always remember to self evaluate your code with SOLID DRY KISS 

You are writing JS directly in HTML, violating the Single Responsibility Principle here.
Olease move all JS code into a separate .js file in /assets/js/

Reply:
Okay. I ll surely update it .